app/assets/javascripts/base-path-preview.js

```diff
@@ -0,0 +1,23 @@
+(function (Modules) {
+  "use strict";
+
+  Modules.BasePathPreview = function () {
+    this.start = function() {
+
+      var basePathWrapper = $('.js-base-path');
+      var basePath = $('.js-base-path input');
+      var pathPrefix = $('select.js-path-prefix');
+      var pathSlug = $('input.js-path-slug');
+
+      pathPrefix.change(updateBasePathPreview);
+      pathSlug.change(updateBasePathPreview);
```

Comment:
What do you think of using ```pathSlug.on('keydown', updateBasePathPreview);```

That way the view-only field gets updated as we type the slug.